Sources/Common/Util/CustomerIOBuilderConfigKeys.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+struct CustomerIOBuilderConfigKeys {
```

Comment:
I am not sure we need these constants. 

From this PR, it looks like the constant values are all used inside of the `SdkConfig.create()` function. If the constants are all encapsulated in 1 place for 1 purpose, I don't see a need for a set of constants like this to maintain. 

I prefer us to remove these constants and have hard-coded strings inside of `SdkConfig` since that's the data type that has the knowledge on how to construct an instance of `SdkConfig`.

Reply:
I am okay with both.

Reply:
I like how you moved the constants into the `SdkConfig` file where they are being used. I didn't consider that idea and I like it. 

I still don't see a need for the constants inside of `SdkConfig` yet, but I am also OK with the solution currently in the PR.